src/Error/Renderer/WebExceptionRenderer.php

```diff
@@ -185,6 +187,11 @@ protected function _getController(): Controller
             $controller = new $class($request);
             $controller->startupProcess();
         } catch (Throwable $e) {
+            Log::warning(
+                "Failed to construct or call startup() on the resolved controller class of `$class`. " .
+                    "Using Fallback Controller instead. Error {$e->getMessage()}",
+                'cake.error'
```

Comment:
New logging has a scope so folks can silence this scope if required. If this is happening frequently in an application I think the application is broken :shrug:

Reply:
Why do we need cake.error if we have log levels? Why not just cake?

Reply:
> Why do we need cake.error if we have log levels? Why not just cake?

I could see a scenario where folks have existing applications that they want to get warnings from but not *this* warning. Having routes based on module/package names felt like a natural pairing with namespaces. I use a similar approach in python applications and it gives you a lot of flexibility in how you can configure logging from your application.